### PR TITLE
chore: deprecate helpers.*Card methods

### PR DESCRIPTION
**Pull request by sprohaszka**
Related to issue #528 

Deprecate the following methods:
* helpers.userCard
* helpers.contextualCard
* helpers.createCard

Replace example in README.

### CODE REVIEW

**Review by ST-DDT — DISMISSED** [review]

**Review by prisis — APPROVED** [review]

File: src/helpers.ts [hunk 1]
```diff
@@ -502,8 +502,12 @@ export class Helpers {
    * //   address: {
    * //     streetA: 'Drake Avenue',
    * // ...
+   * @deprecated If you need a some specific data object you should create your own method.
```
Comment (Shinigami92) [inline src/helpers.ts:505]:
```suggestion
   * @deprecated If you need some specific object you should create your own method.
```

File: src/helpers.ts [hunk 2]
```diff
@@ -502,8 +502,12 @@ export class Helpers {
...
    */
   createCard(): Card {
+    console.warn(
+      'Deprecation Warning: If you need a some specific data object you should create your own method.'
```
Comment (Shinigami92) [inline src/helpers.ts:509]:
```suggestion
      'Deprecation Warning: If you need some specific object you should create your own method.'
```

File: src/helpers.ts [hunk 3]
```diff
@@ -569,8 +573,12 @@ export class Helpers {
    * //   email: 'Eveline.Brekke56.Hoppe@yahoo.com',
    * //   dob: 1964-05-06T05:14:37.874Z,
    * // ...
+   * @deprecated If you need a some specific data object you should create your own method.
```
Comment (Shinigami92) [inline src/helpers.ts:576]:
```suggestion
   * @deprecated If you need some specific object you should create your own method.
```

File: src/helpers.ts [hunk 4]
```diff
@@ -569,8 +573,12 @@ export class Helpers {
...
    */
   contextualCard(): ContextualCard {
+    console.warn(
+      'Deprecation Warning: If you need a some specific data object you should create your own method.'
```
Comment (Shinigami92) [inline src/helpers.ts:580]:
```suggestion
      'Deprecation Warning: If you need some specific object you should create your own method.'
```

File: src/helpers.ts [hunk 5]
```diff
@@ -616,8 +624,12 @@ export class Helpers {
    * //   address: {
    * //     street: 'McKenzie Estates',
    * // ....
+   * @deprecated If you need a some specific data object you should create your own method.
```
Comment (Shinigami92) [inline src/helpers.ts:627]:
```suggestion
   * @deprecated If you need some specific object you should create your own method.
```

File: src/helpers.ts [hunk 6]
```diff
@@ -616,8 +624,12 @@ export class Helpers {
...
    */
   userCard(): UserCard {
+    console.warn(
+      'Deprecation Warning: If you need a some specific data object you should create your own method.'
```
Comment (Shinigami92) [inline src/helpers.ts:631]:
```suggestion
      'Deprecation Warning: If you need some specific object you should create your own method.'
```

**Review by Shinigami92 — CHANGES_REQUESTED** [review]
Could you add also tests to check that the warning is logged out?
You could orient yourself on tests for `random` module.

**Review by Shinigami92 — DISMISSED** [review]

**Review by ST-DDT — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**